packages/context-editor/src/components/AttributePanel.tsx

```diff
@@ -36,66 +36,72 @@ const initPanelProps: PanelProps = {
 	bottom: 0,
 };
 
-registerFormats();
-
 export function AttributePanel({
 	menuHidden,
 	containerRef,
 }: {
 	menuHidden: boolean;
 	containerRef: React.RefObject<HTMLDivElement | null>;
 }) {
-	const [position, setPosition] = useState(initPanelProps);
+	const [offset, setOffset] = useState(initPanelProps);
 	const [height, setHeight] = useState(0);
 	const state = useEditorState();
-	const {
-		activeNode,
-		position: activeNodePosition,
-		setActiveNode,
-		setPosition: setActiveNodePosition,
-	} = useEditorContext();
+	const { position, setPosition } = useEditorContext();
+	const node = position === null ? null : state.doc.nodeAt(position);
```

Comment:
This might be a divisive change, but I removed the `activeNode` state from the editor context, opting to instead only store the position of the opened/active node. I then resolve the active node throughout the editor components using `state.doc.nodeAt()`. This feels a bit more prosemirror-y, and also eliminates a potential source of bugs where the `position` and `activeNode` could get out of sync.